### ISSUE TITLE
fine-tune on the Kitti dataset

### ISSUE THREAD

**Issue by lixiangyu-1008** [djl11/PWC_Net_TensorFlow#3]
hi @djl11 
Thanks for your work. I want to know if I want to fine-tune on the Kitti  Dataset, the ground truth flow are sparse. so how can I change the Loss Function.
